### ISSUE TITLE
Unetbootin does not work in Windows

### ISSUE THREAD

**Issue by Jim-A** [unetbootin/unetbootin#52]
Hi,

I have tried various versions in 3 different PC's and without exception no configuration will recognise the USB stick. Random drive letters appear in the selection box, none of which are valid. 

I am running it under Win7/64 (also 32) SP1 and versions 276, 549, 585 and 613.

This is quite poor considering it is supposed to be a well established product.I don't know anyone else who can get it to work.

I switched to Yumi-1.9.9.9 and that works absolutely fine. So not an urgent fix required, but I can hardly believe how usless unetbootin is.

If you don't believe me or dispute what I am telling you, I can send you a video capture of the procedure. It will be quite large though.

Best Regards,

Jim-A

**Comment by ousia** [comment]
@Jim-A, I just used it in WinXP. So, the statement _UNetbootin does not work in Windows_ is too vague.

It probably does not work with Win7.

**Comment by demonswithin** [comment]
I agree Ousia/ I mean rather than go through an hour video why not give some us your system info such as  cpu speed, whats your ram size, hdd size etc. What brand of PC do you have? A million questions to this one comment.

Im running a test to see if it works with WIndows 10, Ill provide more feed back later.

**Comment by DrVishalKumarSahu** [comment]
UNetBootin does not create a working Live USB of fedora in windows 10. While I was able to create a Live USB of Fedora in Windows 7, 10 days back.
